packages/atomic/src/utils/slot-utils.ts

```diff
@@ -1,23 +1,27 @@
-export function getSlotFromHost(host: HTMLElement, slotName: string) {
+export function getNamedSlotFromHost(host: HTMLElement, slotName: string) {
   const children = Array.from(host.children);
-  const slots = children.filter((child) => {
-    if (slotName) {
-      return child.getAttribute('slot') === slotName;
-    }
+  const targetSlot = children.filter(
+    (child) => child.getAttribute('slot') === slotName
+  );
 
-    return child.getAttribute('slot') === '';
-  });
-
-  if (!slots.length) {
+  if (!targetSlot.length) {
     return;
   }
 
-  if (slots.length > 1) {
-    const msg = slotName
-      ? `Element should only have 1 slot named "${slotName}".`
-      : 'Element should only have 1 default slot.';
-    console.warn(msg, host);
+  if (targetSlot.length > 1) {
+    console.warn(`Element should only have 1 slot named "${slotName}".`, host);
+  }
+
+  return targetSlot[0];
+}
+
+export function getDefaultSlotFromHost(host: HTMLElement) {
+  const children = Array.from(host.children);
+  const defaultSlot = children.filter((child) => !child.hasAttribute('slot'));
+
+  if (defaultSlot.length > 1) {
+    console.warn('Element should only have 1 default slot.', host);
   }
 
-  return slots[0];
+  return defaultSlot[0];
```

Comment:
Is it possible that `defaultSlot` would somehow be an empty array ? And if so should there be a check for this ?